cypress/e2e/publicPaths/render/recoverPassword/index.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { runTestsForDevices } from "../../../utils";
+import { allDevicesForAppLayout } from "../../../utils/devices";
+import { newExpectation, newShouldArgs } from "../../../utils/helpers";
+
+const pageName = "Reset password page";
+const fakeToken = "this-is-a-fake-token";
+const currentPage = `/recover-password?token=${fakeToken}`;
+
+describe(`${pageName} renders expected components on different devices`, () => {
+  const tests = [
+    newExpectation(
+      "should have reset password component with correct title",
+      "[data-cy=reset-password-page]",
+      newShouldArgs("be.visible.and.contain", [
+        "Reset your password",
+        "Please enter a new password below.",
+      ]),
+    ),
+    newExpectation(
+      "should have reset password form with inputs",
+      "[data-cy=reset-password-form] input",
+      newShouldArgs("be.visible.and.have.length", 2),
+    ),
+    newExpectation(
+      "should have new password input",
+      "[data-cy=reset-password-form] div:first-of-type",
+      newShouldArgs("be.visible.and.contain", "New Password"),
```

Comment:
For  `newShouldArgs("be.visible.and.contain", "New Password"),` would it be work using `beVisibleAndContain`
It might be worth it to extend `beVisibleAndContain` to a take and array or just make some `beVisibleAndContains` to take arrays